osl/interface_linux.go

```diff
@@ -62,6 +63,13 @@ func (i *nwIface) Master() string {
 	return i.master
 }
 
+func (i *nwIface) MacAddress() net.HardwareAddr {
```

Comment:
Yes only `osl.Sandbox` interface methods need to be implemented or stubbed for other platforms. Other platforms may not even have a `nwIface`